src/linker/Linker.Dataflow/FlowAnnotations.cs

```diff
@@ -250,7 +250,8 @@ TypeAnnotations BuildTypeAnnotations (TypeDefinition type)
 						}
 
 						if (annotatedMethods.Any (a => a.Method == setMethod)) {
-							_context.LogWarning ($"Trying to propagate DynamicallyAccessedMemberAttribute from property '{property.FullName}' to its setter '{setMethod.GetDisplayName ()}', but it already has such attribute on the 'value' parameter.", 2043, setMethod);
+							_context.LogWarning ($"Trying to propagate DynamicallyAccessedMemberAttribute from property '{property.FullName}' to its setter '{setMethod.GetDisplayName ()}', but it already has such attribute on the 'value' parameter.",
```

Comment:
Should this be DynamicallyAccessedMember**s**Attribute (notice the 's')? If that's the case, this happens in multiple places.

Reply:
Good catch - fixed everwhere.